commander/test/commands/transaction/create/vote.test.ts

```diff
@@ -188,32 +208,43 @@ describe('transaction:create:vote', () => {
 	});
 
 	describe('transaction:create:vote --votes=xxx --unvotes=xxx --no-signature', () => {
```

Comment:
This description also need an update, and same for other cases

Reply:
updated.